queue_and_stack/dll_queue.py

```diff
@@ -7,13 +7,38 @@ class Queue:
     def __init__(self):
         self.size = 0
         # Why is our DLL a good choice to store our elements?
```

Comment:
Was this your question? Or was this written in the exercise? I never know when they change these things around on me haha. 
But to answer the question, DLLs and SLLs are great for storing elements when you don't necessarily have the contiguous memory space to store them. Since each node instance contains pointers to the others, we can store the other nodes in various places in memory. Good question!